src/main/java/de/thi/servicetask/NewUserMessages.java

```diff
@@ -0,0 +1,43 @@
+package de.thi.servicetask;
+
+import io.quarkus.mailer.reactive.ReactiveMailer;
+import io.smallrye.mutiny.Uni;
+import io.quarkus.mailer.Mail;
+import io.quarkus.mailer.Mailer;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class NewUserMessages {
```

Comment:
Es gibt einen Mail-Service. Ihr könnt da mal Jan fragen

Reply:
ja das wurde geschrieben, als es den Mailservice noch nicht gab.
Ich bin gerade dabei den zu implementieren.

Reply:
jedoch nicht in diesem Branch sondern bei -> #31 
(Der Registration Prozess wurde hier entfernt)